src/App.js

```diff
@@ -1,25 +1,66 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from "react";
+import axios from "axios";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      location: {},
+      error: false,
+      errorMessage: '',
+    }
+  }
+
+  handleInput = (e) => {
+    e.preventDefault();
+    this.setState({ searchQuery: e.target.value });
+    console.log(this.state.searchQuery)
```

Comment:
I think this `console.log` will probably print nothing bc `setState` won't have returned yet, but that's a minor point